util/cron/common-python.bash

```diff
@@ -5,33 +5,12 @@
 function set_python_version() {
   local ver_str=$1
 
-  # make sure the length is OK
-  if [[ ${#ver_str} -lt 3 ]] || [[ ${#ver_str} -gt 4 ]]; then
-    echo "Python version string must be in format 'x.y[z]'"
-    return 1
-  fi
-
-  # make sure the 2nd char is "."
-  if [[ "${ver_str:1:1}" != "." ]]; then
-    echo "Python version string must be in format 'x.y[z]'"
-    return 1
-  fi
-
-  local major_ver=${ver_str:0:1}
-  local minor_ver=${ver_str:2}
+  local major_ver=$(echo $ver_str | cut -d. -f1)
+  local minor_ver=$(echo $ver_str | cut -d. -f2)
 
   # override `python`
   export PATH=/hpcdc/project/chapel/no-python:$PATH
 
-  if [[ $major_ver -eq 2 ]]; then
-    # Do some special casing for python2:
-    # 1. Avoid using chpl_make.py and other python scripts as much as possible
-    # 2. Override `python3`
-
-    export CHPL_NIGHTLY_MAKE=gmake
-    export PATH=/hpcdc/project/chapel/no-python3:$PATH
```

Comment:
Oh wow, I wasn't even aware we had that directory.  I imagine we'll delete it once this merges?

Reply:
Me neither. I will go ahead and delete it now since we haven't been testing Python 2.x for a while.

The dir just contains executables that tell you to use `python` rather than `python3`, which I guess makes sense for a Python 2 test. 
```
chapelu@cfljenkins12 chapel $ cd no-python3
chapelu@cfljenkins12 no-python3 $ ls
pip*  pip3*  python3*
chapelu@cfljenkins12 no-python3 $ ./python3
use python2 instead of python3
```